config/locales/interface/slides/en_emissions.yml

```diff
@@ -3,21 +3,26 @@ en:
-      Apart from CO<sub>2</sub> emissions there are also other sources which lead to the emission of greenhouse
-      gases (GHG). These ‘other emissions’ can be separated into 3 groups:
-      <ul><li> Other greenhouse gas emissions which are emitted as a result of energy use, such as methane
-      (CH<sub>4</sub>) or nitrous oxide (N<sub>2</sub>O) </li>
-      <li>CO<sub>2</sub>-emissions of non-energetic activities, such as the use of raw materials in industry
-      or the production of cement</li>
-      <li>Other greenhouse gas emissions from non-energetic activities, such as methane emissions from manure
-      in agriculture</li></ul>
-      <br />In this section these ‘other’ greenhouse gas emissions and non-energetic emissions are displayed.
-      Per sector you can view the amount of other greenhouse gas emissions are emitted as a result of energy use
-      (‘energetic other GHG’), how many non-energetic CO<sub>2</sub> emissions are emitted and how many
-      non-energetic other greenhouse gases.<br /><br />
-      With sliders you can adjust how these emissions will change in the future. You can find more information in
-      our documentation (LINK)."
+      description: |
+        The ETM calculates the amount of CO<sub>2</sub> that is emitted as a result of energy use.
+        Apart from CO<sub>2</sub> emissions there are also other sources which lead to the emission of greenhouse
+        gases (GHG). These ‘other emissions’ can be separated into 3 groups:
+        <ul><li> Other greenhouse gas emissions which are emitted as a result of energy use, such as methane
+        (CH<sub>4</sub>) or nitrous oxide (N<sub>2</sub>O) </li>
+        <li>CO<sub>2</sub>-emissions of non-energetic activities, such as the use of raw materials in industry
+        or the production of cement</li>
+        <li>Other greenhouse gas emissions from non-energetic activities, such as methane emissions from manure
+        in agriculture</li></ul>
+        <br />In this section these ‘other’ greenhouse gas emissions and non-energetic emissions are displayed.
+        Per sector you can view the amount of other greenhouse gas emissions are emitted as a result of energy use
+        (‘energetic other GHG’), how many non-energetic CO<sub>2</sub> emissions are emitted and how many
+        non-energetic other greenhouse gases.<br /><br />
+        With sliders you can adjust how these emissions will change in the future for every sector. 
+        Additionally, you can adjust indirect CO<sub>2</sub>-emissions. These are CO<sub>2</sub>-emissions which are
+        emitted indirectly due to the atmospheric oxidation of methane (CH<sub>4</sub>), carbon monoxide (CO) and 
+        Non-Methane Volatile Organic Compounds (NMVOC) in different sectors.
+        You can find more information in the <a href="https://docs.energytransitionmodel.com/main"
```

Comment:
Better to add a direct link to the relevant article, not the general documentation link.